index.html

```diff
@@ -58,19 +58,21 @@
             </div>
           </div>
         <div class="intro">
-            <iframe 
-                class="hidden"
-                src="http://player.twitch.tv/?channel={TWITCH_USER_NAME}" 
-                height="100%" 
-                width="100%" 
-                frameborder="0" 
-                scrolling="no"
-                allowfullscreen="true"
-            ></iframe>
-          <div class="intro-text">
-            <h1>Twitch Stream offline.</h1>
-            <p>Stream offline. Come back later and watch me play.</p>
-          </div>
+            <script src= "http://player.twitch.tv/js/embed/v1.js"></script>
```

Comment:
I think it would be best to have this down at the end of the file with the other JS `<script>` tags. Probably before the bootstrap include. 

Also using `https` would be awesome.